packages/renderer/src/Loader.svelte

```diff
@@ -64,6 +64,10 @@ window.events.receive('starting-extensions', (value: string) => {
   }
   clearInterval(loadingSequence);
 });
+
+window.events.receive('extensions-started', () => {
```

Comment:
AFAIK extensions-started should not be mapped to 'already started' to avoid confusion

In your usecase it seems that we don't have the readiness of the system ?

Reply:
> AFAIK extensions-started should not be mapped to 'already started' to avoid confusion

Which is the reason behind this?
The system is considered ready when the `starting-extensions` event is received. So we cannot rely on the fact that the system is ready to be sure that the extensions have been already started. We need to wait, and `extensions-started` is the only way we can have to guarantee it, no?

Reply:
@lstocchi the problem that I see is that we could maybe have two events already started being sent
Also there is like a setInterval check. So maybe the condition should wait a longer time and wait remote side say it's ready